assets/data/awards.json

```
@@ -0,0 +1,25 @@
+[
+    {
+        "awards": [
+            {
+                "title": "Most Outstanding Social/Support Organization",
+                "provider": "Anteater Awards 2011"
+            },
+            {
+                "title": "Student Alumni Engagement Sponsorship",
+                "provider": "Student Alumni Association"
+            }
+        ],
+      
+        "recognitions": [
+            { 
+                "title": "Third Place",
+                "provider": "Winter UCI Alumni Association Club Affiliates Challenge 2020"
+            },
+            {
+                "title": "Second Place",
+                "provider": "UCI Campus Organizations & Volunteer Programs | Love My Org Contest 2020"
+            }
+        ]
+    }
+]
```

No need for a list here. It can just be a dictionary with awards and recognitions as its keys
```suggestion
    {
        "awards": [
            {
                "title": "Most Outstanding Social/Support Organization",
                "provider": "Anteater Awards 2011"
            },
            {
                "title": "Student Alumni Engagement Sponsorship",
                "provider": "Student Alumni Association"
            }
        ],
      
        "recognitions": [
            { 
                "title": "Third Place",
                "provider": "Winter UCI Alumni Association Club Affiliates Challenge 2020"
            },
            {
                "title": "Second Place",
                "provider": "UCI Campus Organizations & Volunteer Programs | Love My Org Contest 2020"
            }
        ]
    }
```